### ISSUE TITLE
create `snakemake` wrapper for `msp`?

### ISSUE THREAD

**Issue by andrewkern** [tskit-dev/msprime#2104]
TIL that `snakemake` now has features "tool wrappers" which allows popular command line tools to be used more simply with `snakemake` workflows: https://snakemake.readthedocs.io/en/stable/tutorial/additional_features.html#tool-wrappers

I wonder if users would like an `msp` wrapper?

**Comment by jeromekelleher** [comment]
Nice! The ``msp`` cli doesn't do an awful lot though, so I'm not sure what workflows we'd be simplifying?

**Comment by andrewkern** [comment]
yeah, agreed! but maybe it would help _someone_? obviously not high priority
